rtc_data_enclave/src/lib.rs

```diff
@@ -55,3 +55,14 @@ pub unsafe extern "C" fn rtc_validate_and_save(
         (err) => EcallResult::Err(DataUploadError::Sealing(err)),
     }
 }
+
+/// Tries to perform local attestation to an enclave at dest_enclave_id. The enclave needs to be
+/// initialized as a `ResponderSys` in rtc_udh'
```

Comment:
(May be better to link to `DhSessions::establish_new` as the main documented function here?)

```suggestion
/// Tries to perform local attestation to an enclave at dest_enclave_id.
///
/// See: [`DhSessions::establish_new`]
```